File: abc161/b/b.php
<?php
[$n, $m] = explode(" ", trim(fgets(STDIN)));
$a = explode(" ", trim(fgets(STDIN)));
$check = 0;
$total = array_sum($a);
for($i = 0; $i < $n; $i++){
  if($a[$i] * 4 * $m >= $total){
    $check ++;
  }
}
if($check >= $m){
  echo "Yes\n";
}
else{
  echo "No\n";
}

?>

File: abc161/b/b.go
package main

import "fmt"

func vote(n int, m int, a []int, total int) bool {
	check := 0

	for i := 0; i < n; i++ {
		if a[i]*4*m >= total {
			check++
			if check == m {
				return true
			}
		}
	}
	return false
}

func main() {
	var n, m int
	fmt.Scan(&n, &m)
	a := make([]int, n, n+1)
	total := 0
	for i := range n {
		fmt.Scan(&a[i])
		total += a[i]
	}
	if vote(n, m, a, total) {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}
